jake/__main__.py

```diff
@@ -32,7 +32,10 @@ def main():
   """jake entry point"""
   args = __add_parser_args_and_return()
   log = __setup_logger(args.verbose)
-  config = Config()
+  if args.application:
+    config = IQConfig()
+  else:
+    config = Config()
 
   if args.snake:
     __get_config_from_std_in(config)
```

Comment:
Minor: down on line 43, I think the: `config = IQConfig()` of:

```
elif args.python:
    config = IQConfig()
```
is now obsolete, and should be deleted.